desktop/core/ext-py3/boto-2.49.0/boto/gs/connection.py

```diff
@@ -39,12 +39,12 @@ def __init__(self, gs_access_key_id=None, gs_secret_access_key=None,
                  proxy_user=None, proxy_pass=None,
                  host=DefaultHost, debug=0, https_connection_factory=None,
                  calling_format=SubdomainCallingFormat(), path='/',
-                 suppress_consec_slashes=True):
+                 suppress_consec_slashes=True, anon=False):
         super(GSConnection, self).__init__(gs_access_key_id, gs_secret_access_key,
                  is_secure, port, proxy, proxy_port, proxy_user, proxy_pass,
                  host, debug, https_connection_factory, calling_format, path,
                  "google", Bucket,
-                 suppress_consec_slashes=suppress_consec_slashes)
+                 suppress_consec_slashes=suppress_consec_slashes, anon=anon)
```

Comment:
What is anon stands for?

Reply:
Anonymous auth. We set this param to True when RAZ is available so that auth is done by signed headers sent by RAZ.

This is similar to S3 implementation.